### PR TITLE
Origin/tests for markerref classes

### PR DESCRIPTION
**Pull request by mykola-pysarev**

### CODE REVIEW

File: xmind/tests/test_markerref_markerid.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+from xmind.core.markerref import MarkerId
+import logging
+from . import base
+
+
+class MarkerrefTest(base.Base):
+    """Markerref test"""
+    LOGGER = logging.getLogger('MarkerefTest')
+
+    def test_init_set_name(self, name='test-name'):
```
Comment (andrii-z4i) [inline xmind/tests/test_markerref_markerid.py]: What is the reason to have test_ function with parameter? I would declare either just variable inside the method or even would go without any variable for such simple test.
Like 
```
def test_bla_bla_bla(self): 
el = MarkerId('some')
self.assertEqual(el.name, 'some')```

**Review by andrii-z4i — CHANGES_REQUESTED** [review]
It's awesome! But let's discuss the necessity to have parametrized test functions before we do merge.

File: xmind/tests/test_markerref_markerid.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        el = MarkerId(_name)
+        self.assertEqual(el.__str__(), _name)
+
+    def test_repr_method(self):
+        """test that __repr__ method exists and returns correct representation"""
+        _name = 'test-name'
+        _el = MarkerId(_name)
+        self.assertEqual(_el.__repr__(), "<MarkerId: %s>" % _name)
+
+    def test_get_family_method(self):
```
Comment (andrii-z4i) [inline xmind/tests/test_markerref_markerid.py:42]: Please add one test for getFamily for the case when our 'name' doesn't have dash '-', like ```testname```

**Review by andrii-z4i — CHANGES_REQUESTED** [review]

File: xmind/tests/test_markerref_markerid.py (hunk 2 above)
Comment (mykola-pysarev) [reply xmind/tests/test_markerref_markerid.py:42]: Have added.

**Review by andrii-z4i — APPROVED** [review]
Ready to go to master!

**Comment by andrii-z4i** [comment]
@mykola-pysarev please push 'squash and merge' button but rename title by removing 'origin/' from it.

**Pull request merged**